### PR TITLE
travis: update cache

### PR DESCRIPTION
**Pull request by BusyJay**
Cache more things to speedup travis build.

### CODE REVIEW

**Comment by siddontang** [comment]
CI failed

**Comment by BusyJay** [comment]
PTAL

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,12 @@ sudo: false
...
 cache:
   directories:
-  - $HOME/.cache/
-  - $HOME/.cargo/
+    - $HOME/.cache/
+    - $HOME/.cargo/
+    - $TRAVIS_BUILD_DIR/target
+
+before_cache:
+  - find $TRAVIS_BUILD_DIR/target/debug -maxdepth 1 -type f -delete
```
Comment (siddontang) [inline .travis.yml]: no need to care /target/release?

Comment (BusyJay) [inline .travis.yml]: No, only debug target is enabled when building in travis.

**Comment by AndreMouche** [comment]
LGTM

**Pull request merged**